src/main/java/org/dita/dost/ant/types/JobSourceSet.java

```diff
@@ -58,9 +59,9 @@ public JobSourceSet() {
             for (final FileInfo f : job.getFileInfo(this::filter)) {
                 log("Scanning for " + f.file.getPath(), Project.MSG_VERBOSE);
                 final File tempFile = new File(job.tempDir, f.file.getPath());
-                if (tempFile.exists()) {
+                if (job.getStore().exists(tempFile.toURI())) {
```

Comment:
All cases where we used to test file existance with `File.exists()`, we use `Store.exists(URI)` instead. The default `StreamStore` implementation will just test if the file exists. The added abstraction here allows in-memory `Store` to first check if the resource exists in memory, then fall back to checking if the resource exists in disk.